src/Presentation/settings.default.json

```diff
@@ -0,0 +1,8 @@
+{
+  "environmentVariables": {
+    "ASPNETCORE_ENVIRONMENT": "Development"
+  },
```

Comment:
Do we really need to create a `environmentVariables` section here? Won't it be enough just to have this?
```json
"environment": "Development"
```